src/pair.c

```diff
@@ -1088,14 +1097,35 @@ int get_pair_mapping(char* config, PAIR* p, int n)
         {
             //it's  used, check if it is  conditioned
             float scale=1, offset=0;
-            if(-1 == get_pair_arg_conditioning(tmp, var, &scale, &offset))//get conditioning for midi arg
+            if(-1 == get_pair_arg_conditioning(tmp, var, &scale, &offset))//get conditioning for osc arg
             {
-                printf("\nERROR in config line:\n%s could not get OSC arg conditioning for arg %s!\n\n",config,tmp);
+                printf("\nERROR in config line:\n%s -could not get OSC arg conditioning for arg %s!\n\n",config,tmp);
```

Comment:
I think that the formatting is ok, I only added the dash there since it's in the other messages, too.